docs/sphinx/_ext/sphinx-mdx-builder/sphinxcontrib/mdxbuilder/writers/mdx.py

```diff
@@ -183,6 +183,8 @@ def end_state(
         end: Sequence[str] | None = ("",),
         first: str | None = None,
     ) -> None:
+        if len(self.stateindent) == 0:
```

Comment:
There's some odd RST files that are causing weird issues with indenting, this is here to prevent us from trying to pop an empty list